src/app/authentication/authentication.component.ts

```diff
@@ -1,8 +1,126 @@
-import { Component } from '@angular/core';
+import { Component, ElementRef, OnInit, ViewChild } from '@angular/core';
+import { Router } from '@angular/router';
+import { AuthenticationService } from '../authentication.service';
 
 @Component({
     selector: 'app-authentication',
     templateUrl: './authentication.component.html',
     styleUrls: ['./authentication.component.css']
 })
-export class AuthenticationComponent {}
+export class AuthenticationComponent implements OnInit {
+    title: string = 'Penda';
```

Comment:
Is this line needed here?

Reply:
I think this is some visual bug because on my end it's all good